### PR TITLE
Updated kubeone URL

### PR DESCRIPTION
**Pull request by pnutmath**

### CODE REVIEW

**Comment by tengqm** [comment]
@pnutmath Please consider signing the CLA in order for this to be reviewed and possibly merged.

**Comment by pnutmath** [comment]
> @pnutmath Please consider signing the CLA in order for this to be reviewed and possibly merged.

Done with signing CLA

File: content/en/docs/setup/_index.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ The following production environment solutions table lists the providers and the
 | [IBM](https://www.ibm.com/in-en/cloud)        | [IBM Cloud Kubernetes Service](https://cloud.ibm.com/kubernetes/catalog/cluster)| |[IBM Cloud Private](https://www.ibm.com/in-en/cloud/private) | |
 | [Ionos](https://www.ionos.com/enterprise-cloud)        | [Ionos Managed Kubernetes](https://www.ionos.com/enterprise-cloud/managed-kubernetes) | [Ionos Enterprise Cloud](https://www.ionos.com/enterprise-cloud) | |
 | [Kontena Pharos](https://www.kontena.io/pharos/)          |    |&#x2714;|    &#x2714;    |        |         |
-| [KubeOne](https://github.com/kubermatic/kubeone) |  | &#x2714; | &#x2714; | &#x2714; | &#x2714; | &#x2714; |
```
Comment (RA489) [inline content/en/docs/setup/_index.md]: This url is working fine for kubermatic/kubeone repo

File: content/en/docs/setup/_index.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ The following production environment solutions table lists the providers and the
...
+| [KubeOne](https://kubeone.io/) |  | &#x2714; | &#x2714; | &#x2714; | &#x2714; | &#x2714; |
```
Comment (RA489) [inline content/en/docs/setup/_index.md]: Any specific reason changing to this url?

Comment (pnutmath) [reply content/en/docs/setup/_index.md]: Yeah... Existing URL too working.. 

Just to get directly to website as other links have

**Pull request merged**